ppx/ppx_deriving_json/lib/ppx_deriving_json.ml

```diff
@@ -79,9 +78,9 @@ let int ?loc ?attrs x = Exp.constant ?loc ?attrs (Pconst_integer (string_of_int
 
 let pint ?loc ?attrs x = Pat.constant ?loc ?attrs (Pconst_integer (string_of_int x, None))
 
-let lid ?(loc = !default_loc) s = Location.mkloc (parse_lid s) loc
+let lid ?(loc = !default_loc) s = { txt = parse_lid s; loc }
```

Comment:
What about introducing two functions `mkloc` and `mknoloc` as a way to make the diff smaller ?

Reply:
I've introduced the function `mkloc` now. What are you referring to with `mknoloc`?

Reply:
Ok, I've seen now that you were referring to file `ppx/ppx_js/lib_internal/ppx_js_internal.ml `.